exports.js

```diff
@@ -441,6 +441,7 @@ module.exports = {
         'snsCrossAccount'               : require(__dirname + '/plugins/aws/sns/snsCrossAccount.js'),
         'snsValidSubscribers'           : require(__dirname + '/plugins/aws/sns/snsValidSubscribers.js'),
         'snsTopicHasTags'               : require(__dirname + '/plugins/aws/sns/snsTopicHasTags.js'),
+        'snsTopicnoHttpPolicy'          : require(__dirname + '/plugins/aws/sns/snsTopicnoHttpPolicy.js'),
```

Comment:
Change filename to snsTopicNoHttpPolicy
```suggestion
        'snsTopicnNHttpPolicy'          : require(__dirname + '/plugins/aws/sns/snsTopicNoHttpPolicy.js'),
```